packages/spotlib/spotlib.4.2.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+synopsis: "Useful functions for OCaml programming used by @camlspotter"
+description: """\
+Spotlib is a small library package used for several softwares by Jun Furuse.
+It is almost a poor replication of Jane Street Core, but it is small."""
+maintainer: "jun.furuse@gmail.com"
+authors: "Jun Furuse"
+homepage: "https://gitlab.com/camlspotter/spotlib"
+bug-reports: "https://gitlab.com/camlspotter/spotlib/-/issues"
+depends: [
+  "dune" {build & >= "2.0"}
+  "ocaml" {>= "4.08.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.08.0" & < "5.0"}
```
```
#=== ERROR while compiling spotlib.4.2.0 ======================================#
# context              2.1.2 | linux/x86_64 | ocaml-variants.5.0.0+trunk | pinned(https://gitlab.com/camlspotter/spotlib/-/archive/4.2.0/spotlib-4.2.0.tar.gz)
# path                 ~/.opam/5.0/.opam-switch/build/spotlib.4.2.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p spotlib -j 31
# exit-code            1
# env-file             ~/.opam/log/spotlib-3577-337b84.env
# output-file          ~/.opam/log/spotlib-3577-337b84.out
### output ###
# (cd _build/default && /home/opam/.opam/5.0/bin/ocamlc.opt -w -3-9 -g -bin-annot -I lib/.spotlib.objs/byte -I /home/opam/.opam/5.0/lib/ocaml/str -I /home/opam/.opam/5.0/lib/ocaml/unix -I /home/opam/.opam/5.0/lib/ppx_test -I /home/opam/.opam/5.0/lib/ppx_test/runtime -I /home/opam/.opam/5.0/lib/re -I /home/opam/.opam/5.0/lib/re/pcre -I /home/opam/.opam/5.0/lib/seq -no-alias-deps -open Spotlib -o lib/.spotlib.objs/byte/spotlib__Xformat.cmi -c -intf lib/xformat.pp.mli)
# File "lib/xformat.mli", line 61, characters 39-62:
# 61 | val set_formatter_tag_functions : t -> formatter_tag_functions -> unit
#                                             ^^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound type constructor formatter_tag_functions
# Hint: Did you mean formatter_stag_functions?
```

Reply:
Do you want to fix that before releasing or is the constraint good enough for now?